indico/modules/events/timetable/templates/display/balloons/block.html

```diff
@@ -0,0 +1,88 @@
+{% from 'attachments/_management_info_column.html' import render_attachment_info %}
+
+<div class="balloon">
+    <div class="title-container session-block">
+        <div class="title">
+            {{- block.session.title }}
+            {%- if block.title -%}
+                <span class="balloon-subtitle"> ({{ block.title }})</span>
```

Comment:
I'd get rid of the space before `(` and simply not strip spaces in front of the `if block.title`. this looks like something very tempting for someone to accidentally "fix" in the future, resulting in `title(blocktitle)`